cmd/post.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"path"
+
+	"github.com/spf13/cobra"
+)
+
+type Payload struct {
+	Content string `json:"content"`
+}
+
+var postCmd = &cobra.Command{
+	Use:   "post",
+	Short: "Post text to discord server",
+	Run: func(cmd *cobra.Command, args []string) {
+		home, _ := os.UserHomeDir()
+		configPath := path.Join(home, ".config/dobato/webhook")
+
+		if _, err := os.Stat(configPath); errors.Is(err, os.ErrNotExist) {
+			fmt.Println("No config available, run dobato setup first.")
+			return
+		}
```

Comment:
need to check another errors

Reply:
[d046f70](https://github.com/himkt/dobato-go/pull/1/commits/d046f70acbaf09311ee3e0193aa06d6a0411e394) 👈